Project.toml

```diff
@@ -14,7 +14,7 @@ SymEngine_jll = "3428059b-622b-5399-b16f-d347a77089a4"
 Compat = "0.63.0, 1, 2, 3"
 RecipesBase = "0.1, 0.2, 0.3, 0.4, 0.5, 0.6, 0.7, 0.8, 1.0"
 SpecialFunctions = "0.3, 0.4, 0.5, 0.6, 0.7, 0.8, 0.9, 0.10, 1, 2"
-SymEngine_jll = "0.8"
+SymEngine_jll = "0.9"
 julia = "1.3"
```

Comment:
Need to update this to `1.6` and then update the CI too

Reply:
Updated this PR and created #253 for CI

Reply:
Thanks. Can you rebase on top of master now?